eng/performance/benchmark_jobs.yml

```diff
@@ -26,7 +26,9 @@ jobs:
             value: '--dotnet-versions %DOTNET_VERSION% --cli-source-info args --cli-branch %PERFLAB_BRANCH% --cli-commit-sha %PERFLAB_HASH% --cli-repository https://github.com/%PERFLAB_REPO% --cli-source-timestamp %PERFLAB_BUILDTIMESTAMP%'
           - name: AdditionalHelixPreCommands
             value: 'call %HELIX_CORRELATION_PAYLOAD%\tools\machine-setup.cmd'
-          - name: Python
+          - name: PythonJob
```

Comment:
@billwert  Kindly remind me is this the variable name you'd prefer to change?  (the name's lost in our Teams chat)

Reply:
yeah, I would have left this just as `Python`. `PythonBuild` is the special one for the CI build machine, and `Python` can just be everywhere else.

Reply:
Yup thanks for reminding me!